File: huntkingdom2/src/user/Entity/Annonce.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package user.Entity;

import javafx.scene.image.ImageView;

/**
 *
 * @author belkis
 */
public class Annonce {
    private int id_annonce;
    private String text;
    private String image;
    private user user;
    private int aime; 
    private String a;
  private ImageView img; 
  private int etat;

    public void setEtat(int etat) {
        this.etat = etat;
    }

    public int getEtat() {
        return etat;
    }

    public Annonce(int id_annonce, user user, int etat) {
        this.id_annonce = id_annonce;
        this.user = user;
        this.etat = etat;
    }

    public Annonce(int id_annonce, user user, int aime, int etat) {
        this.id_annonce = id_annonce;
        this.user = user;
        this.aime = aime;
        this.etat = etat;
    }

    public Annonce(int id_annonce, String text, String image, String a, int aime, ImageView img) {
        this.id_annonce = id_annonce;
        this.text = text;
        this.image = image;
         this.a = a;
        this.aime = aime;
       
        this.img = img;
    }

    public void setImg(ImageView img) {
        this.img = img;
    }

    public ImageView getImg() {
        return img;
    }

    
  
    public Annonce(int id_annonce, String text, String image, int aime, String a) {
        this.id_annonce = id_annonce;
        this.text = text;
        this.image = image;
        this.aime = aime;
        this.a = a;
    }

    public void setA(String a) {
        this.a = a;
    }

    public String getA() {
        return a;
    }

    public Annonce() {
    }

    public Annonce(int id_annonce, int aime) {
        this.id_annonce = id_annonce;
        this.aime = aime;
    }

    public Annonce(String text, String image) {
        this.text = text;
        this.image = image;
    }

    public Annonce(String text, String image, user user, int aime) {
        this.text = text;
        this.image = image;
        this.user = user;
        this.aime = aime;
    }

    public Annonce(int id_annonce, String text, String image, user user, int aime) {
        this.id_annonce = id_annonce;
        this.text = text;
        this.image = image;
        this.user = user;
        this.aime = aime;
    }

    public void setAime(int aime) {
        this.aime = aime;
    }

    public int getAime() {
        return aime;
    }

    public Annonce(int id_annonce, String text, String image, user user) {
        this.id_annonce = id_annonce;
        this.text = text;
        this.image = image;
        this.user = user;
    }

    public Annonce(String text, String image, user user) {
        this.text = text;
        this.image = image;
        this.user = user;
    }

    public int getId_annonce() {
        return id_annonce;
    }

    public String getText() {
        return text;
    }

    public String getImage() {
        return image;
    }

    public user getUser() {
        return user;
    }

    public void setId_annonce(int id_annonce) {
        this.id_annonce = id_annonce;
    }

    public void setText(String text) {
        this.text = text;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public void setUser(user user) {
        this.user = user;
    }

    public Annonce(int id_annonce, String text, String image, user user, int aime, String a, int etat) {
        this.id_annonce = id_annonce;
        this.text = text;
        this.image = image;
        this.user = user;
        this.aime = aime;
        this.a = a;
        this.etat = etat;
    }

    @Override
    public String toString() {
        return "Annonce{" + "id_annonce=" + id_annonce + ", text=" + text + ", image=" + image + ", user=" + user + ", aime=" + aime + ", a=" + a + ", img=" + img + ", etat=" + etat + '}';
    }

  
    
    
    
    
    
    

   
}

File: symfony/project/sympfony/src/HuntkingdomBundle/Entity/Annonce.php
<?php

namespace HuntkingdomBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Annonce
 *
 * @ORM\Table(name="annonce", indexes={@ORM\Index(name="user_id", columns={"user_id"})})
 * @ORM\Entity
 */
class Annonce
{
    /**
     * @return string
     */
    public function getText()
    {
        return $this->text;
    }

    /**
     * @param string $text
     */
    public function setText($text)
    {
        $this->text = $text;
    }

    /**
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * @param string $image
     */
    public function setImage($image)
    {
        $this->image = $image;
    }

    /**
     * @return int
     */
    public function getAime()
    {
        return $this->aime;
    }

    /**
     * @param int $aime
     */
    public function setAime($aime)
    {
        $this->aime = $aime;
    }

    /**
     * @return int
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * @param int $etat
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;
    }

    /**
     * @return User
     */
    public function getUser()
    {
        return $this->user;
    }

    /**
     * @param User $user
     */
    public function setUser($user)
    {
        $this->user = $user;
    }
    /**
     * @var integer
     *
     * @ORM\Column(name="id_annonce", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idAnnonce;

    /**
     * @var string
     *
     * @ORM\Column(name="text", type="string", length=255, nullable=false)
     */
    private $text;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255, nullable=false)
     */
    private $image;

    /**
     * @var integer
     *
     * @ORM\Column(name="aime", type="integer", nullable=true)
     */
    private $aime;

    /**
     * @var integer
     *
     * @ORM\Column(name="etat", type="integer", nullable=false)
     */
    private $etat;

    /**
     * @return int
     */
    public function getIdAnnonce()
    {
        return $this->idAnnonce;
    }

    /**
     * @param int $idAnnonce
     */
    public function setIdAnnonce($idAnnonce)
    {
        $this->idAnnonce = $idAnnonce;

    }

    /**
     * @var \HuntkingdomBundle\Entity\User
     *
     * @ORM\ManyToOne(targetEntity="HuntkingdomBundle\Entity\User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="user_id", referencedColumnName="id_user")
     * })
     */
    private $user;


}

